public/lib/composer-service.js

```diff
@@ -148,12 +148,12 @@ function wfComposerService($http, $q, config, $log, wfHttpSessionService, wfTele
 
         const tags = {
             contentType: getType(type),
-            displayHint: selectedDisplayHint,
-            commissionedLength,
             productionOffice: prodOffice,
-            commissioningDesk: commissioningDeskExternalName,
             priority: getPriorityName(priority),
         }
+        if(selectedDisplayHint) tags.displayHint = selectedDisplayHint;
```

Comment:
Generally nicer to explicitly check for not null and not undefined rather than coercing to a boolean. Particularly relevant for commissionedLength, when you would want to send zero if it is zero.

Reply:
I'm with you. Let me change that.